src/sentry/options/defaults.py

```diff
@@ -1999,6 +1999,12 @@
     default=14,
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "statistical_detectors.query.functions.timeseries_days",
```

Comment:
the option for transactions already exist above, just replicating it for functions too